pkg/api/validation/validation_test.go

```diff
@@ -8609,6 +8609,71 @@ func getStorageResourceList(storage string) api.ResourceList {
 	return res
 }
 
+func getLocalStorageResourceList(ephemeralStorage string) api.ResourceList {
```

Comment:
Can u reuse `getLocalStorageResourceList` in validation_test.go and admission_test.go ?

Reply:
I wrote them in two files just like `getStorageResourceList`, and was not sure whether we need to put them into a utility package.